libraries/AC_AutoTune/AC_AutoTune.cpp

```diff
@@ -644,9 +644,6 @@ void AC_AutoTune::control_attitude()
             twitching_test_rate(rotation_rate, target_rate, test_rate_min, test_rate_max);
             twitching_measure_acceleration(test_accel_max, rotation_rate, rate_max);
             twitching_abort_rate(lean_angle, rotation_rate, abort_angle, test_rate_min);
-            if (lean_angle >= target_angle) {
```

Comment:
I guess the removal of the "lean_angle >= target_angle" check is intentional and not necessary because the equivalent check is inside the "twitching_test_rate" function?

Reply:
Yes that is correct.